### ISSUE TITLE
Dash extension: Open URL without activating app

### ISSUE THREAD

**Issue by Kapeli** [pilotmoon/PopClip-Extensions#634]
The current PopClip Dash extension opens a `dash://` URL. The problem is that the system forces Dash to become active when this URL is opened.

Currently, this is not an issue. However, a future update of Dash will add support for iOS remotes. While an iOS remote is connected, extensions are redirected to it directly and Dash on OS X is not shown at all. Unfortunately there's no way for me to prevent the system from activating Dash when you open the `dash://` URL.

I assume PopClip uses something like this right now:
```objc
[[NSWorkspace sharedWorkspace] openURL:dashURL];
```

Is there any way to get PopClip to do this instead?
```objc
NSTask *dash = [[NSTask alloc] init];
[dash setLaunchPath:@"/usr/bin/open"];
[dash setArguments:@[@"-g", @"dash://query"]];
[dash launch];
```

The `-g` causes Dash to not get activated if it does not want to and the user experience is a lot better as the active app does not lose focus anymore.

**Comment by invariant** [comment]
Yes this one's pretty easy. The easiest way is to switch to a shell script extension (see the referenced commit). I will push this out main branch, but can you confirm if this behaviour using -g will be OK with the current version of Dash, or do I need to wait for an update?

**Comment by Kapeli** [comment]
Looks great.  Thanks for the change! Using `-g` has no effect on the current and previous versions of Dash. They all know to activate themselves if they need to.
